K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -90,6 +90,40 @@ public static void DeleteValuesToComparePercentiles(JToken parent)
             }
         }
 
+        /// <summary>
+        /// Get all descendants tokens from a root token
+        /// </summary>
+        public static IEnumerable<JToken> GetAllDescendantsTokens(JToken rootToken)
+        {
+            var toSearch = new Stack<JToken>(rootToken.Children());
+            while (toSearch.Count > 0)
+            {
+                var inspected = toSearch.Pop();
+                yield return inspected;
+                foreach (var child in inspected)
+                {
+                    toSearch.Push(child);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Make a Math.Round operation on all float values in a json payload.
+        /// </summary>
+        public static void RoundFloats(JToken parent, string jsonPath, int? digits = null)
+        {
+            if (digits.HasValue)
```

Comment:
if (!digits.HasValue){
  return;
}

Reply:
In my projects I like to use return as soon as possible.
But in most of the projects I've been working so far, developers don't like the return key word (for some reasons)

But if it's something you are comfortable with, I'm in 100% !